imports/api/NewGameUtils.js

```diff
@@ -47,60 +47,58 @@ export var NewGameUtils = {
 
 		// This code could be streamlined, but let's leave it explicit
 		switch (gameType) {
-		case Constants.GAME_TYPE.HONG_KONG:
-			if (round <= 4)
-				return "東";
-			if (round > 4 && round <= 8)
-				return "南";
-			if (round > 8 && round <= 12)
-				return "西";
-			else //if (round > 12)
-				return "北";
-			break;
-		case Constants.GAME_TYPE.JAPANESE:
-			if (round <= 4)
-				return "東";
-			if (round > 4 && round <= 8)
-				return "南";
-			else //if (round > 8)
-				return "西";;
-			break;
+			case Constants.GAME_TYPE.HONG_KONG:
+				if (round <= 4)
+					return "東";
+				if (round > 4 && round <= 8)
+					return "南";
+				if (round > 8 && round <= 12)
+					return "西";
+				else //if (round > 12)
+					return "北";
+			case Constants.GAME_TYPE.JAPANESE:
+				if (round <= 4)
+					return "東";
+				if (round > 4 && round <= 8)
+					return "南";
+				else //if (round > 8)
+					return "西";
 		};
 	},
 
 	// Helper function to ensure all players are selected
 	allPlayersSelected() {
-		return (Session.get("current_east") != Constants.DEFAULT_EAST && 
```

Comment:
This is just the style I've been taught, but can formatting please be put in a seperate branch if you think it's important?  Otherwise there's just a lot of clutter.